### DISCUSSION TITLE
Seamoth and Hand Questions

### DISCUSSION THREAD

**Discussion by Phearos** [Okabintaro/SubmersedVR#94] [Q&A]
Hi, great work on the mod, this game is perfect for VR.  When I built the Seamoth I was saying wow a lot, but when jumping in found myself unable to turn with the thumb sticks (pitching is fine), how do you turn the Seamoth?  Also, is it possible to make the left hand dominate?

**Comment by NeonPixels** [comment]
I personally had no trouble controlling the seamoth.
I use the PICO 4, maybe my default bindings work correctly? Try looking at the SteamVR bindings for Subnautica, although I've personally had a hard time understanding how they work.

**Comment by Phearos** [comment]
I'm on the Quest 2 and it's the same with the camera from the scanner room, pitching no problem but can't turn left right.  I've tried changing the bindings by adding anything that's listed as left or right to the thumb stick, but nothing worked.  My workaround is to pitch straight down which causes the seamoth to keep turning till I raise it up and I have the direction I need.
